modules/op-release-notes-1-0.adoc

```diff
@@ -72,6 +72,38 @@ For this release, use flags `disable-home-env-overwrite` and `disable-working-di
 == Known issues
 * If you are upgrading from an older version of {pipelines-title}, you must delete your existing deployments before upgrading to {pipelines-title} version 1.0. To delete an existing deployment, you must first delete Custom Resources and then uninstall the {pipelines-title} Operator. For more details, see the uninstalling {pipelines-title} section.
 * Submitting the same `v1alpha1` Tasks more than once results in an error. Use `oc replace` instead of `oc apply` when re-submitting a `v1alpha1` Task.
+* `buildah` ClusterTask does not work on the {pipelines-title} version 1.0.
++
+During the Operator installation, the `--storage-driver` flag for the `buildah` ClusterTask is set to `default` value which causes the `/etc/containers/storage.conf` settings to be ignored. This results in the failure of `buildah` ClusterTask.
++
+As a workaround, you can manually set the `--storage-driver` flag to `overlay` value in the `buildah-task.yaml` file:
++
+. Login to your cluster as a `cluster-admin`:
++
+----
+$ oc login -u <login> -p <password> https://openshift.example.com:6443
+----
+. Open the `buildah-task.yaml` file for editing:
```

Comment:
```suggestion
. Edit the `clustertask/` `buildah` via the `oc edit` editing:
```

Reply:
How about - . Use the `oc edit` command to edit `buildah` ClusterTask